package.json

```diff
@@ -159,6 +158,7 @@
     "json-loader": "0.5.7",
     "lint-staged": "8.1.4",
     "lodash.camelcase": "4.3.0",
+    "lodash.mapvalues": "4.6.0",
```

Comment:
Do we still need it or not? 🤔

Reply:
It's in the story...

Reply:
https://github.com/commercetools/ui-kit/blob/2861a1b289f2e5c05f57d7bbd2f1d7a950b95dd6/examples/form-inputs.example.story.js#L10